frontend/src/components/authentication/RegistrationForm.jsx

```diff
@@ -970,10 +970,24 @@ class RegistrationForm extends Component {
           isBackdropStatic={true}
           show={this.state.showPrivacyNoticeDialog}
           handleClose={this.closePrivacyNoticePopup}
-          title={"TITLE HERE (TODO)"}
+          title={LOCALIZE.authentication.createAccount.privacyNoticeDialog.title}
           description={
             <div>
-              <p>DESCRIPTION HERE (TODO)</p>
+              <h3>
+                {LOCALIZE.authentication.createAccount.privacyNoticeDialog.privacyNoticeStatement}
+              </h3>
```

Comment:
You should add `<section></section>` for each < h3 > tag.

`<section aria-labelledby="<id>">`
`<h3 id="<id>">...</h3>`
`<p>...</p>`
`</section>`

In that case, screen reader users would be able to skip sections if they want to.

Reply:
Oops somehow I missed this - I will follow up with a PR that also includes the links!